apps/portal/src/components/pages/SupportPage.js

```diff
@@ -8,12 +8,12 @@ const SupportPage = () => {
     const [isLoading, setLoading] = useState(true);
     const [error, setError] = useState(null);
     const [disabledFeatureError, setDisabledFeatureError] = useState(null);
-    const {member, t} = useContext(AppContext);
+    const {member, t, site} = useContext(AppContext);
 
     useEffect(() => {
         async function checkoutDonation() {
-            const siteUrl = window.location.origin;
-            const currentUrl = siteUrl + window.location.pathname;
+            const siteUrl = site.url;
+            const currentUrl = window.location.href.split('#')[0];
```

Comment:
I'd suggest keeping `const currentUrl = window.location.origin + window.location.pathname`, so that both hash fragments and query parameters are stripped from the resulting URL.